keyboards/hazel/bad_wings/matrix.c

```diff
@@ -0,0 +1,137 @@
+// Copyright 2022 @sadekbaroudi (Sadek Baroudi)
```

Comment:
this should probably have your name below sadek's

Reply:
Updating to include myself.